src/NHibernate/Loader/OuterJoinableAssociation.cs

```diff
@@ -228,16 +229,17 @@ internal string GetSelectFragment(string entitySuffix, string collectionSuffix,
 
 				case SelectMode.FetchLazyPropertyGroup:
 					return ReflectHelper.CastOrThrow<ISupportLazyPropsJoinable>(Joinable, "fetch lazy property")
-					                    .SelectFragment(
-						                    next?.Joinable,
-						                    next?.RHSAlias,
-						                    RHSAlias,
-						                    collectionSuffix,
-						                    ShouldFetchCollectionPersister(),
-						                    new EntityLoadInfo(entitySuffix)
-						                    {
-							                    LazyProperties = EntityFetchLazyProperties
-						                    });
+										.SelectFragment(
+											next?.Joinable,
+											next?.RHSAlias,
+											RHSAlias,
+											collectionSuffix,
+											ShouldFetchCollectionPersister(),
+											new EntityLoadInfo(entitySuffix)
+											{
+												LazyProperties = EntityFetchLazyProperties, 
+												IncludeLazyProps = SelectMode == SelectMode.FetchLazyProperties,
+											});
```

Comment:
Wider code change than expected, due to some bad interaction between VS formatting and JetBrains formatting while using tabs, I think.
For this reason, I tend to prefer writing:
```
return aThing
	.DoSomething()
	.DoSomethingElse();
```
Rather than:
```
return aThing.DoSomething()
             .DoSomethingElse();
```
The former does not change on reformat, while the later tends to get wrecked on reformat.

Anyway here, may you either revert to previous formatting, or if changing it, change for a more stable one?

Reply:
restored previous formatting